bridgepoint/prebuild.py

```diff
@@ -1755,19 +1755,18 @@ def accept_BodyNode(self, node):
     
     
 class RequiredOperationPrebuilder(ActionPrebuilder):
-    element_type = "Required Operaton"
+    element_type = 'Required Operaton'
+    label = None
+    
     def __init__(self, metamodel, spr_ro):
         self._spr_ro = spr_ro
         c_c = one(spr_ro).SPR_REP[4502].C_R[4500].C_IR[4009].C_PO[4016].C_C[4010]()
         c_i = one(spr_ro).SPR_REP[4502].C_R[4500].C_IR[4009].C_I[4012]()
         c_po = one(spr_ro).SPR_REP[4502].C_R[4500].C_IR[4009].C_PO[4016]()
-        self._path = get_path(c_c, c_c.Name + '::' + c_po.Name + '::' + c_i.Name + '::' + spr_ro.Name)
+        self.label = '::'.join([get_parent_label(c_c), c_c.Name, c_po.Name,
+                                c_i.Name, spr_ro.Name])
```

Comment:
why is the interface name part of the label?

Reply:
It matches the hierarchy in the model tree, which allows locating the problem element easier.